### ISSUE TITLE
Support for non Magisk super user in ReVanced Manager

### ISSUE THREAD

**Issue by Relected** [ReVancedTeam/revanced-suggestions#83]
i used to have an issue with Vanced where i can't install root-variant with other alternative superuser managers such as phh superuser or superSU

vanced root variant was magis specific, which was not good cuz of their not-so-good libsu implementation

can this not happen with revanced?

**Comment by ps1-hagrid** [comment]
I would only question the security and integrity of the project. As the previous commit was nearly 5 years ago.

**Comment by ps1-hagrid** [comment]
Yeah no reason to use alternative root solutions unless you're on Android 4.4.4. Magisk is the way in the current world.

**Comment by oSumAtrIX** [comment]
> I personally agree with ps1-hagrid. Don't you think it's time to upgrade? Superuser might be outdated, full of bugs, and/or contain vulnerabilities.

I have consulted him, he is forced to use these alternatives due to his phone

**Comment by ps1-hagrid** [comment]
Well there seems to be no solution as all the alternate SU libraries are very out of date.

**Comment by oSumAtrIX** [comment]
There is a solution, but it's not elegant. Technically you can execute all the commands via a system call. Basically doing `system("su -c CMD")`. If anyone wants, they can implement an alternate way like that along side the current way.

**Comment by ps1-hagrid** [comment]
Sounds quite janky. But if there is request for it. It should be there.
